src/FilterPluginManagerFactory.php

```diff
@@ -6,26 +6,22 @@
 
 use Interop\Container\ContainerInterface;
 use Laminas\ServiceManager\Config;
-use Laminas\ServiceManager\FactoryInterface;
-use Laminas\ServiceManager\ServiceLocatorInterface;
+use Laminas\ServiceManager\Factory\FactoryInterface;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\NotFoundExceptionInterface;
 
 use function is_array;
 
 class FilterPluginManagerFactory implements FactoryInterface
 {
-    /**
-     * laminas-servicemanager v2 support for invocation options.
-     *
-     * @param array
-     */
-    protected $creationOptions;
```

Comment:
A note, in case somebody gets to this PR and wonders why these properties and methods are removed: It looks like we've not supported laminas-servicemanager v2 for some time now, so the v2 properties and methods are dead code.

(Likely should have called that out in the PR description; as it was, I had to verify this was the case during review.)

Reply:
Understood. I will keep this change strictly to the bugfix